gateway/modules/crud/bolt/raw.go

```diff
@@ -0,0 +1,26 @@
+package bolt
+
+import "context"
+
+func (b *Bolt) RawExec(ctx context.Context, project string) error {
+	return nil
+}
+
+func (b *Bolt) CreateProjectIfNotExist(ctx context.Context, project string) error {
+	return nil
+}
+
+func (b *Bolt) RawBatch(ctx context.Context, batchedQueries []string) error {
+	return nil
+}
+
+// GetConnectionState : function to check connection state
+func (b *Bolt) GetConnectionState(ctx context.Context) bool {
+	if !b.enabled || b.client == nil {
+		return false
+	}
+
+	// Ping to check if connection is established
+	err := b.connect()
```

Comment:
Is there a better way to do this?

Reply:
![image](https://user-images.githubusercontent.com/24411676/72267138-9180c480-3645-11ea-9bb0-ffeacc73fbdb.png)
we do have some utility function which can be used, if succesfully executed we can assume it is client is still connected with the database